### PR TITLE
deployment fixes

### PR DESCRIPTION
**Pull request by SamuelGabriel**
### Change Description

*Try to be precise. You can additionally add comments to your PR, this might help the reviewer a lot.*

Just tiny fixes for deployment.

**If you used new dependencies: Did you add them to `requirements.txt`?**

**Who did you ping on Mattermost to review your PR? Please ping that person again whenever you are ready for another review.**
@liam-sbhoo @anshulg954 

## Breaking changes

If you made any breaking changes, please update the version number.
Breaking changes are totally fine, we just need to make sure to keep the users informed and the server in sync.

**Does this PR break the API? If so, what is the corresponding server commit?**
no

**Does this PR break the user interface? If so, why?**
no
---
*Please do not mark comments/conversations as resolved unless you are the assigned reviewer. This helps maintain clarity during the review process.*

### CODE REVIEW

**Review by anshulg954 — APPROVED** [review]
LGTM

File: pyproject.toml [hunk 1]
```diff
@@ -4,13 +4,12 @@ build-backend = "hatchling.build"
 
 [project]
 name = "tabpfn-client"
-version = "0.0.14"
+version = "0.0.15"
```
Comment (liam-sbhoo) [inline pyproject.toml:7]: Could you update PROD config in google bucket after merging this? :)

**Review by liam-sbhoo — APPROVED** [review]
Just a reminder to update PROD config in the google bucket when this is merged. Thanks!

File: pyproject.toml (hunk 1 above)
Comment (SamuelGabriel) [reply pyproject.toml:7]: I will try to do it tomorrow :)

**Pull request merged**